schema-registry/common/src/main/java/com/hortonworks/registries/schemaregistry/retry/policy/ExponentialBackoffRetryPolicy.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2016-2019 Cloudera, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hortonworks.registries.schemaregistry.retry.policy;
+
+import com.google.common.annotations.VisibleForTesting;
+
+import java.util.Map;
+
+public class ExponentialBackoffRetryPolicy extends RetryPolicy {
+
+    public static final String EXPONENT = "exponent";
+
+    private static final long DEFAULT_SLEEP_TIME_MS = 1000L;
+    private static final float DEFAULT_EXPONENT = 2;
+    private static final int DEFAULT_MAX_RETRIES = 10;
+    private static final long DEFAULT_MAX_SLEEP_TIME_MS = 90_000L;
```

Comment:
The default value contradicts itself. Either increase the DEFAULT_MAX_SLEEP_TIME_MS (or) reduce the #retries to 6.

Reply:
I have reduced it to 5